src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Query/SqliteAllPersistenceIdsSpec.cs

```diff
@@ -15,7 +15,7 @@ namespace Akka.Persistence.Sqlite.Tests.Query
 {
     public class SqliteAllPersistenceIdsSpec : AllPersistenceIdsSpec
     {
-        public static readonly AtomicCounter Counter = new AtomicCounter(50);
+        public static readonly AtomicCounter Counter = new AtomicCounter(100);
```

Comment:
What's with all of the `AtomicCounter` value changes in these specs? Not criticizing, just wondering.